appserver/transaction/jts/src/main/java/com/sun/enterprise/transaction/jts/iiop/TransactionServerInterceptor.java

```diff
@@ -16,18 +17,20 @@
 
 package com.sun.enterprise.transaction.jts.iiop;
 
-import org.omg.CORBA.LocalObject;
-import org.omg.PortableInterceptor.ServerRequestInterceptor;
-import org.omg.PortableInterceptor.ServerRequestInfo;
-
 import org.glassfish.enterprise.iiop.api.GlassFishORBHelper;
 import org.glassfish.hk2.api.ServiceLocator;
+import org.omg.CORBA.LocalObject;
+import org.omg.PortableInterceptor.ServerRequestInfo;
+import org.omg.PortableInterceptor.ServerRequestInterceptor;
 
 import com.sun.enterprise.transaction.api.JavaEETransactionManager;
 
-public class TransactionServerInterceptor extends LocalObject
-        implements ServerRequestInterceptor, Comparable {
+public class TransactionServerInterceptor extends LocalObject implements ServerRequestInterceptor, Comparable {
 
+    /**
```

Comment:
Please, remove this unnecessary comment.

Reply:
thx!